### PR TITLE
OSS featuring, sorting, and authoring

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

cuddling lions

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

- [I want "featured" OSS projects](https://trello.com/c/dGgqSrQ9)
- [Don't need to show "oddbird" projects on emeritus-bird pages](https://trello.com/c/nHKy1mKo) 

## Description
_The commit messages say what you did; this should explain why and/or how._

- [x] Featured projects are sorted to the top, and get a full summary description
- [x] I changed it so we're currently only using 'oddbird' as the author on truly full-team projects (so far, only Herman). We could take that a step farther by checking to see if the emeritus bird overlapped with the project? But at this point, that's only Rachel, and she doesn't have a page anyway…
- [x] Since Claudina helped create Accoutrement, I added her to that (and added her face sketch to the page)

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- /open-source/
- /authors/claudina/
- (all author pages)

### CODE REVIEW

**Comment by mirisuzanne** [comment]
[Don't need to show "oddbird" projects on emeritus-bird pages](https://trello.com/c/nHKy1mKo/59-dont-need-to-show-oddbird-projects-on-emeritus-bird-pages)

**Comment by mirisuzanne** [comment]
[I want "featured" OSS projects](https://trello.com/c/dGgqSrQ9/58-i-want-featured-oss-projects)

File: .eleventy.js [hunk 1]
```diff
@@ -54,7 +54,26 @@ module.exports = (eleventyConfig) => {
...
+        }
+
+        if (a.data.end !== b.data.end) {
+          if (a.data.end === 'ongoing') {
+            return -1;
+          }
+          if (b.data.end === 'ongoing') {
+            return 1;
+          }
+          return b.data.end - a.data.end;
```
Comment (jgerigmeyer) [inline .eleventy.js:72]: Do we need to check whether `a.data.end` and `b.data.end` exist before doing this? It looks like some OSS pages (e.g. sass, true, herman, helium, django, css-remedy) don't have `end` defined at all.

File: content/_includes/birds.macros.njk [hunk 2]
```diff
@@ -40,12 +40,14 @@ params:
...
-      {% set tween = ' & ' | mdInline if (loop.length == 2) else ',' %}
+      {{ '&' if loop.last and (loop.length > 2) }}
       {{ author(
         name=name,
         collections=collections,
-        link=link
+        link=link,
+        fullname=false
       ) }}
+      {%- set tween = ' & ' | mdInline if (loop.length == 2) else ',' -%}
```
Comment (jgerigmeyer) [inline content/_includes/birds.macros.njk:50]: Why does this one go through `mdInline` but the earlier one doesn't?

File: content/_includes/birds.macros.njk [hunk 3]
```diff
@@ -76,11 +78,14 @@ params:
...
-  link=true
+  link=true,
+  fullname=true
 ) -%}
   {%- set page = collections.birds | authorPage(name) if (collections and not page) else page -%}
   {%- if page.data -%}
-    {%- set name = page.data.title if (name != 'oddbird') else 'OddBird' -%}
+    {%- set all = 'OddBird' if (name == 'oddbird') else none -%}
+    {%- set bird = page.data.title if fullname else (page.data.shortname or name | capitalize) -%}
+    {%- set name =  all or bird -%}
```
Comment (jgerigmeyer) [inline content/_includes/birds.macros.njk:88]:
```suggestion
    {%- set name = all or bird -%}
```

File: content/_data/taxonomy.yaml [hunk 4]
```diff
@@ -18,14 +18,21 @@ post:
...
       - maintainer
       - invited expert
   - title: Other Contributions
-    show: contributor
+    all:
+      - contributor
+    past:
+      - core team
+      - maintainer
+      - invited expert
```
Comment (jgerigmeyer) [inline content/_data/taxonomy.yaml:34]: This structure seems confusing to me... but I can't think of a quick better way 🤷‍♂️

File: .eleventy.js (hunk 1 above)
Comment (mirisuzanne) [reply .eleventy.js:72]: Seems like that would imply the same as "ongoing" - so we could maybe remove the explicit version, and then replace the earlier lines with eg…

`if (b.data.end === 'ongoing') { … }` => `if (!b.data.end) { … }`

File: content/_includes/birds.macros.njk (hunk 2 above)
Comment (mirisuzanne) [reply content/_includes/birds.macros.njk:50]: oh, I bet we added that initially to convert `&` -> `<span class="amp">&</span>` with typogr. I'll look into either doing that with both, or I think there might be a simpler way to handle our ampersand typography…

File: content/_data/taxonomy.yaml (hunk 4 above)
Comment (mirisuzanne) [reply content/_data/taxonomy.yaml:34]: yeah, I tried several ways of arranging it… ¯\\\_(ツ)_/¯

File: content/_includes/birds.macros.njk (hunk 2 above)
Comment (jgerigmeyer) [reply content/_includes/birds.macros.njk:50]: What's weird is that it looked to me like both of those ampersands were already being converted anyway 🤷

File: content/_includes/birds.macros.njk [hunk 5]
```diff
@@ -40,13 +40,16 @@ params:
...
         name=name,
         collections=collections,
-        link=link
+        link=link,
+        fullname=false
       ) }}
-      {{- tween | safe if not loop.last }}
+      {%- if not loop.last -%}
+        {{ ' & ' if loop.length == 2 else ', ' }}
+      {%- endif -%}
```
Comment (jgerigmeyer) [inline content/_includes/birds.macros.njk:52]: @mirisuzanne The `mdInline | safe` tags don't seem to be needed here 🤷

**Review by jgerigmeyer — APPROVED** [review]

File: content/_includes/birds.macros.njk (hunk 5 above)
Comment (jgerigmeyer) [reply content/_includes/birds.macros.njk:52]: @mirisuzanne I think this is back to you to merge if [my commit](https://github.com/oddbird/oddleventy/pull/143/commits/2e5b30abf374ca05dcd4e46bed306ed0a5a2da79) looks okay.

**Pull request merged**